tests/unit/test_lookups.py

```diff
@@ -0,0 +1,230 @@
+import pytest
+import os
+import csv
+
+from digital_land.pipeline import Lookups
+
+
+@pytest.fixture
+def pipeline_dir(tmp_path):
+    pipeline_dir = os.path.join(tmp_path, "pipeline")
+    os.makedirs(pipeline_dir, exist_ok=True)
+
+    # create lookups
+    row = {
+        "prefix": "ancient-woodland",
+        "resource": "",
+        "organisation": "local-authority-eng:ABC",
+        "reference": "ABC_0001",
+        "entity": "1234567",
+    }
+    fieldnames = row.keys()
+
+    with open(os.path.join(pipeline_dir, "lookup.csv"), "w") as f:
+        dictwriter = csv.DictWriter(f, fieldnames=fieldnames)
+        dictwriter.writeheader()
+        dictwriter.writerow(row)
+
+    return pipeline_dir
+
+
+@pytest.fixture
+def empty_pipeline_dir(tmp_path):
+    pipeline_dir = os.path.join(tmp_path, "pipeline")
+    os.makedirs(pipeline_dir, exist_ok=True)
+
+    return pipeline_dir
+
+
+def test_lookups_load_csv_success(
+    pipeline_dir,
+):
+    """
+    test csv loading functionality
+    :param pipeline_dir:
+    :return:
+    """
+    lookups = Lookups(pipeline_dir)
+    lookups.load_csv()
+
+    expected_num_entries = 1
+    assert len(lookups.entries) == expected_num_entries
+
+
+def test_lookups_load_csv_failure(
+    empty_pipeline_dir,
+):
+    """
+    test csv loading functionality when file absent
+    :param pipeline_dir:
+    :return:
+    """
+    lookups = Lookups(empty_pipeline_dir)
+
+    with pytest.raises(FileNotFoundError) as fnf_err:
+        lookups.load_csv()
+
+    expected_exception = "No such file or directory"
+    assert expected_exception in str(fnf_err.value)
+
+
```

Comment:
should be able to run a test that doesn't need a file system by just adding them to the class